.golangci.yml

```
@@ -0,0 +1,49 @@
+service:
+  golangci-lint-version: 1.21.0 # Keep this in sync with .circleci/config.yml and Jenkinsfile.pr
+
+run:
+  timeout: 5m
+  modules-download-mode: readonly
+
+linters-settings:
+  gofmt:
+    simplify: true
+  govet:
+    check-shadowing: true
+    enable-all: true
+
```

I usually set `golint` to a more aggressive setting and have not encountered any false positives yet.

```suggestion
  golint:
    min-confidence: 0

```